### DISCUSSION TITLE
Github Actions/Workflows in mobile app

### DISCUSSION THREAD

**Discussion by fischer-felix** [community/community#35561] [Mobile] [answered]
Are there plans to support Actions or Workflows in the GitHub android app?

**Comment by The-Noah** [comment, marked as answer]
It seems they just added it as a feature, but only for PRs and not the entire repo? https://github.blog/changelog/2022-10-04-introducing-actions-on-github-mobile/

**Reply by trackpadpro** [reply]
I have been waiting for this for quite a while. Happy to see they've been working on it.
